viam-orb-slam3.go

```diff
@@ -258,8 +250,8 @@ func configureCameras(ctx context.Context, svcConfig *slamConfig.AttrConfig, dep
 
 // Position forwards the request for positional data to the slam library's gRPC service. Once a response is received,
 // it is unpacked into a PoseInFrame.
-func (slamSvc *builtIn) Position(ctx context.Context, name string, extra map[string]interface{}) (*referenceframe.PoseInFrame, error) {
-	ctx, span := trace.StartSpan(ctx, "slam::builtIn::Position")
+func (orbSvc *orbslamService) Position(ctx context.Context, name string, extra map[string]interface{}) (*referenceframe.PoseInFrame, error) {
+	ctx, span := trace.StartSpan(ctx, "slam::orbslamService::Position")
```

Comment:
[q] I wonder if we should rename this to this? (I'd need to do the same)

```suggestion
	ctx, span := trace.StartSpan(ctx, "slam::orbslam::Position")
```

Reply:
Do you mean we should change the name of the struct `orbslamService` into `orbslam` or that we should change these trace lines?

Reply:
Just the trace lines. But maybe I'm wrong! Let's discuss in person.

Reply:
We have decided to go with `viamorbslam3::orbslamService::Position`